Release Standard Operating Procedures (SOP) (#3100)
grafana/agent: docs/developer/release/5-update-release-branch.md

# Update Release Branch

> The `release` branch is a special branch that is used for grafana cloud to point at our install scripts and example kubernetes manifests. This is not to be confused with `release-[version prefix]` created in [Create Release Branch](./1-create-release-branch.md)

Reviewer: ```suggestion
The `release` branch is a special branch that is used for grafana cloud to point at our install scripts and example kubernetes manifests. This is not to be confused with `release-RELEASE_VERSION_PREFIX` created in [Create Release Branch](./1-create-release-branch.md)
```

## Steps

1. Force push the release tag to the `release` branch

    ```
    git fetch
    git checkout main

>     git branch -f release [release version]

Reviewer: ```suggestion
    git branch -f release RELEASE_VERSION
```

    git push -f origin refs/heads/release
    ```

>     *NOTE: This requires force push permissions on this branch so you may need an assist doing this step*

Reviewer: ```suggestion
    > **NOTE**: This requires force push permissions on this branch. If this fails, reach out to one of the project maintainers for help. 
```
